lib/temporal/workflow/history/event.rb

```diff
@@ -10,8 +10,6 @@ class Event
           ACTIVITY_TASK_CANCELED
           TIMER_FIRED
           REQUEST_CANCEL_EXTERNAL_WORKFLOW_EXECUTION_FAILED
-          WORKFLOW_EXECUTION_SIGNALED
-          WORKFLOW_EXECUTION_TERMINATED
```

Comment:
Why were these removed?

Reply:
The constant is only used as part of this case statement and those two event types don't have the `scheduled_event_id` unlike the others in that list so they don't feel like they belong.

Reply:
It's fine for them to stay if you'd prefer now terminated event is correctly "shadowed" by the proceeding `when`. Just not really sure what determines which events should be in the list.

Reply:
Ah, I see, good point 👍